src/Security/Permission.php

```diff
@@ -350,26 +357,15 @@ public static function groupList($memberID = null)
         }
 
         if ($member) {
-            // Build a list of the IDs of the groups.  Most of the heavy lifting
-            // is done by Member::Groups
-            // NOTE: This isn't efficient; but it's called once per session so
-            // it's a low priority to fix.
-            $groups = $member->Groups();
-            $groupList = [];
-
-            if ($groups) {
-                foreach ($groups as $group) {
-                    $groupList[] = $group->ID;
-                }
-            }
-
+            // Build a list of the IDs of the groups.
+            $groupList = $member->Groups()->column();
```

Comment:
```suggestion
            $groupIDs = $member->Groups()->column();
```
Also rename the variable in this method

Reply:
Done